index.js

```diff
@@ -0,0 +1,118 @@
+const tripsUrl = 'https://trektravel.herokuapp.com/trips';
+
+const displayStatus = (message) => {
+    $('#status').html(message);
+  };
+  
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p><ul>`;
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+const showTripDetails = id => {
+  const detailsUrl = tripsUrl + `/${id}`;
+  axios.get(detailsUrl)
+    .then(function(response) {
+        const trip = response.data;
+        let info = `<h3> ${trip.name}</h2>
+        <p>${trip.about}</p>
+        <ul>
+        <li><b>Continent</b>: ${trip.continent}</li>
+        <li><b>Category</b>: ${trip.category}</li>
```

Comment:
I don't know that I would use a `<ul>` here - what you're displaying isn't quite a list.